packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -137,6 +142,16 @@ export const PayableDetailsForm = forwardRef<
   </MoniteScopedProviders>
 ));
 
+export const defaultRequiredFields = {
+  invoiceNumber: true,
+  dueDate: true,
+  tags: true,
+  currency: true,
+  invoiceDate: false,
```

Comment:
Fields with 'false' can be dropped in code since they aren't required and this won't affect the code behavior.